js/tests/unit/dropdown.js

```diff
@@ -498,6 +498,39 @@ $(function () {
     $dropdown.trigger('click')
   })
 
+  QUnit.test('should fire hide and hidden event with a mouseEvent', function (assert) {
```

Comment:
I you want to add this property only for click event, you should add a test which verify this property isn't added on other type of events